src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalController.java

```diff
@@ -133,26 +148,35 @@ public ResponseEntity findUserByEmail(@RequestParam(value = "email") String emai
     @ApiResponses({
             @ApiResponse(
                     code = 201,
-                    message = "User Roles has been added",
+                    message = "The User's Roles have been modified",
                     response = OrganisationResponse.class
             ),
+            @ApiResponse(
+                    code = 400,
+                    message = "Invalid request provided"
+            ),
             @ApiResponse(
                     code = 403,
                     message = "Forbidden Error: Access denied"
             ),

```

Comment:
I think we are not only modifying the roles here status as well so include that(applies for both internal and external)
code = 412 missing in the internal end point

Reply:
Will make these changes

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/503/commits/adb2277a233c24a0c2520c68efc956d0635c322a